docs: dark brand (#1614)
quarto-dev/quarto-web: docs/authoring/brand.qmd

::: 

#### Document logo customization

> The document `logo` option allows extra customization of the logo.

Reviewer: ```suggestion
You can further customize the logo at the document-level with the `logo` option.
```

> To choose a brand logo resource:

Reviewer: ```suggestion
For example, you can specify a particular brand logo resource:
```

```{.yaml filename="document.qmd"}
---
logo: large
---
```

> To choose a brand logo resource and change the alt text:

Reviewer: ```suggestion
Or, specify a brand logo and change the alt text:
```

```{.yaml filename="document.qmd"}
---
logo:
  path: large
  alt: Alternate alternate text
---
```

::: {.callout-warning}

## Limitation

It is [not currently possible](https://github.com/quarto-dev/quarto-cli/issues/11309) to customize the logo for a specific page within a website. Only the project's `_brand.yml` is applied.

::: 

The Typst implementation allows customization of the logo position at the document level:

> ```{.yaml filename="document.qmd"}
> ---
> logo:
>   width: 1in
>   location: right-top
>   padding-right: 0.5in
>   padding-top: 0.25in
>   alt: Alternate alternate text
> ---
> ```

Reviewer: I think we should specifically target `typst` in the YAML of this example. This?
````suggestion
```{.yaml filename="document.qmd"}
---
format: typst
logo:
  width: 1in
  location: right-top
  padding-right: 0.5in
  padding-top: 0.25in
  alt: Alternate alternate text
---
```
````
Or maybe nest `logo` under `typst` to really emphasize these are `typst` only options.

Author: I like nesting it under `typst`! I had to test it to make sure it works. I'll do that.
